eisbuk-admin/src/components/atoms/AttendanceCard/__tests__/AttendanceCard.test.tsx

```diff
@@ -54,10 +68,61 @@ describe("AttendanceCard", () => {
     });
     test("should disable attendance button while state and fb are synching", () => {
       screen.getByText("👎").click();
-      expect(screen.getByTestId("SaulGoodman")).toHaveProperty(
+      expect(screen.getByTestId("Saul-Goodman")).toHaveProperty(
         "disabled",
         true
       );
     });
+    /** @TODO find a good test for options */
+    test("should render intervals in dropdown when athlete is present", () => {
+      screen.getByText("👎").click();
+      expect(screen.getByTestId("Saul-Goodman-input")).toBeInTheDocument();
+
+      // expect(screen.getAllByTestId("SaulGoodman13:00 - 14:00").length).toEqual(
+      //   customersSlot.intervals.length
+      // );
+    });
+    test("should render bookedInterval value as display value", () => {
+      screen.getByText("👎").click();
+      expect(screen.getByTestId("Saul-Goodman-select")).toBeInTheDocument();
+      const firstOption = screen.getByTestId("Saul-Goodman-select").firstChild;
+      expect(firstOption).toHaveTextContent("13:00 - 14:00");
+    });
+    test("should allow for selecting multiple intervals", () => {
+      fireEvent.change(screen.getByTestId("Saul-Goodman-input"), {
+        target: { value: "13:15 - 14:15" },
+      });
+      expect(screen.getByTestId("Saul-Goodman-input")).toHaveDisplayValue([
+        "13:00 - 14:00",
+      ]);
+    });
+    test("should allow for deselecting intervals", () => {
+      fireEvent.change(screen.getByTestId("Saul-Goodman-input"), {
+        target: { value: "13:15 - 14:15" },
+      });
+      fireEvent.change(screen.getByTestId("Saul-Goodman-input"), {
+        target: { value: "13:15 - 14:15" },
+      });
+      expect(screen.getByTestId("Saul-Goodman-input")).toHaveDisplayValue([
+        "13:00 - 14:00",
+      ]);
+    });
```

Comment:
This become obsolete when the test/functionality is changed to select only one interval.